lib/mongo/operation/specifiable.rb

```diff
@@ -332,6 +337,18 @@ def options
       # @since 2.2.0
       def read_concern
         spec[READ_CONCERN]
+
+      # Whether or not to bypass document level validation.
+      #
+      # @example Get the bypass_document_validation option.
+      #
+      # @return [ Boolean ] Whether to bypass document level validation.
+      #
+      # @since 3.2.0 (?)
+      end
```

Comment:
same here: 2.2.0

I think you put the comments in the read_concern method by accident.